conference/models.py

```diff
@@ -1308,7 +1308,12 @@ class NoQuestionsAvailable(Exception):
         pass
 
     question = models.CharField(max_length=255)
-    answer   = models.CharField(max_length=255)
+    answer   = models.CharField(max_length=255,
+                                verbose_name='''answer (use a regular expression
+                                to capture possible answers e.g.
```

Comment:
It's better to use separate string literals on each line, than using a long string with lots of spaces inside.

Reply:
I didn't know this, thank you. It should be corrected in the new change.